hadoop-ozone/ozone-manager/pom.xml

```diff
@@ -271,6 +271,44 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
           <excludeFilterFile>${basedir}/dev-support/findbugsExcludeFile.xml</excludeFilterFile>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>depcheck</id>
+            <phase></phase>
+          </execution>
+          <execution>
+            <id>banned-rocksdb-imports</id>
+            <phase>process-sources</phase>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <RestrictImports>
+                  <includeTestCode>false</includeTestCode>
+                  <reason>Use managed RocksObjects under org.apache.hadoop.hdds.utils.db.managed instead.</reason>
+                  <!-- By default, ban all the classes in org.rocksdb -->
+                  <bannedImport>org.rocksdb.**</bannedImport>
+                  <allowedImports>
+                    <!-- Allow these imports for snapshot diff. We can't use managed DB for Snapshot diff because
```

Comment:
well, if that's true, you can add those snapshot diff classes in a package and put it under `exclusion`.

Reply:
You can't have multiple exclusion rules unfortunately.

Reply:
Actually,  the `managed` package can always be extended to adapt to new needs. And it's important for this feature to use managed RocksObject to assert memory leak.

Reply:
Thanks @duongkame for the suggestion. Updated to use Managed objects.